README.md

```diff
@@ -40,9 +40,9 @@ viewing this README from GitHub.com, assuming a repo context might be best, wher
 Alternatively, if this package is distributed on PyPi, perhaps just ``pip install <package-name>``, followed by quick
 user instructions, will suffice.
 
-  * ``git clone https://github.com/ssec-jhu/base-template``
-  * ``conda create -n package_name python pip``
-  * ``conda activate package_name``
+  * ``git clone git@github.com:ssec-jhu/flfm-ij-plugin.git``
```

Comment:
[revision] This is using the ssh opt which would require the docs to explain setting that up but we don't so the default http ver is better.